docs/architecture/adrs/007-orchestrator-telemetry.md

```diff
@@ -0,0 +1,74 @@
+and outgoing requests.
+
+### Metrics
+The orchestrator will aggregate metrics regarding the requests it has received/handled, and annotate the metrics with
+span attributes allowing for detailed filtering and monitoring. The metrics will be exported through the OTLP exporter
+through the metrics provider. Traces exported through the traces provider can also have R.E.D. (request, error and
+duration) metrics attached to them implicitly by the OTEL collector using the `spanmetrics` connector. Both the OTLP
+metrics and the `spanmetrics` metrics can be exported to configured metrics backends like Prometheus or Grafana.
+The orchestrator will handle a variety of useful metrics such as counters and histograms for received/handled 
+successful/failed requests, request and stream durations, and server/client errors.
+
+### Configuration
+The orchestrator will expose CLI args/env variables for configuring the OTLP exporter endpoint(s) 
+(e.g. `OTEL_EXPORTER_OTLP_ENDPOINT=http://localhost:4317`) and protocol(s) (e.g. `OTEL_EXPORTER_OTLP_PROTOCOL=grpc`), 
+as well as the ability to enable trace (includes logs) and metrics OTLP exporting (e.g. `OTLP_EXPORT=traces,metrics`.
+Alongside this, the orchestrator will also allow for configuring the Rust log level (e.g. `RUST_LOG=debug`) and 
+log format to use for logs printed to stdout (e.g. `STDOUT_LOG=pretty`). Printing logs to stdout can also be disabled
+(e.g. `STDOUT_LOG=none`). The OTEL collector can be configured to receive from the OTLP endpoint(s) and export to
+observability backends. Examples of a complete configuration for the orchestrator and OTEL collector can be found in
+the playbooks [LINK HERE] and [LINK HERE] as Docker Compose files and as an OpenShift deployment.
+
+### Cross-service tracing
+The orchestrator and client services will be able to consolidate telemetry and share observability through a common
+configuration and backends. This will be made possible through the use of the OTLP standard as well as through the
+propagation of the trace context through requests across services using the standardized `traceparent` header. The
+orchestrator will make use of an incoming request's `traceparent` for initializing a trace context for a request if
+it exists, or else will initialize a new trace. 
+
+## Status
+
+Proposed
+
+## Consequences
+
+- The orchestrator and client services have a common standard to conform to for telemetry, allowing for traceability
+  across different services.
+- The orchestrator and client services do not need to be concerned with specific observability backends, the OTEL
+  collector and OTLP standard can be used to export telemetry data to a variety of backends including Jaeger,
+  Prometheus, Grafana, and Instana, as well to OpenShift natively through the OpenTelemetryCollector CRD.
+- Using the `tracing` crate in Rust for logging will treat logs as traces, allowing the orchestrator to export logs
```

Comment:
Can you also add another line if there is any implication to stdout logging with this PR (in comparison to what exist today)?